.ci/bwcVersions

```diff
@@ -55,3 +55,4 @@ BWC_VERSION:
   - "2.3.0"
```

Comment:
@dreamer-89 we need 1.3.8 here as well (if I am not mistaken)

Reply:
Thanks @reta . Updated.